src/Compilers/CSharp/Test/Semantic/Semantics/RefLocalsAndReturnsTests.cs

```diff
@@ -637,9 +637,9 @@ void M(ref string s)
     }
 }");
             comp.VerifyDiagnostics(
-                // (8,17): error CS1510: A ref or out value must be an assignable variable
+                // (8,17): error CS0206: A property or indexer may not be passed as an out or ref parameter
```

Comment:
>as an out or ref parameter [](http://example.com/codeflow?start=81&length=26)

I do not see `s2` used as a parameter.

Reply:
The property `s2` is passed as an out parameter `s`, the error message says it correctly.

Reply:
> The property ```s2``` is passed as an out parameter s

Sorry, I do not see an out parameter `s` in this scenario. And I do not see `s2` passed as an argument to any method. What am I missing?

Reply:
You're right, I got confused, because `s` is an `out` parameter of the containing method and `s2` is "passed as output" from that method.